docs/guides/screen-reader.md

```diff
@@ -119,45 +121,57 @@ onDragEnd = (result: DropResult, provided: HookProvided) => {
 }
 ```
 
-This announcement should:
+**Default message**: "Movement cancelled. The item has returned to its starting position of ${result.source.index + 1}"
 
-- Inform the user that the drag have been cancelled
-- Let the user know where the item has returned to
+We tell the user the following:
 
-**Default message**: "Movement cancelled. The item has returned to its starting position of ${result.source.index + 1}"
+- The drag has been cancelled
+- Where the item has returned to
+
+Think about adding information about the length of the list, and the name of the list you have dropped into.
 
-You are also welcome to add information about the size of the list, and the name of the list you have dropped into.
+**Message with more info**: "Movement cancelled. The item has returned to list `${listName}` to its starting position of `${startPosition}` of` ${listLength}`".
 
-**Suggestion** "Movement cancelled. The item has returned to list ${listName} to its starting position of ${startPosition} of ${listLength}".
+#### Scenario 2. Dropped in the home list (in new position)
 
-### Drop in the home list
+**Default message**: "You have dropped the item. It has moved from position `${result.source.index + 1}` to `${result.destination.index + 1}`"
 
 This announcement should:
```

Comment:
Do we need this line if we've got "We tell the user the following:" below? I'd get rid of it.

Reply:
I must have missed this. I'll remove it